perl/sieve/scripts/installsieve.pl

```diff
@@ -1,4 +1,6 @@
 #! /bin/sh
+exec perl -x -S $0 ${1+"$@"} # -*-perl-*-
+#!perl -w
 # SPDX-License-Identifier: BSD-3-Clause-CMU
```

Comment:
This change looks like it's fixing an overzealous deletion from earlier, but was squashed with the wrong commit?